apps/ima/behaviours/harm-claim-summary.js

```diff
@@ -0,0 +1,59 @@
+const _ = require('lodash');
+
+module.exports = superclass => class extends superclass {
+  configure(req, res, next) {
+    if(req.sessionModel.get('sih-countries') && !req.sessionModel.get('sih-countries').aggregatedValues.length) {
+      req.form.options.addStep = 'harm-claim';
+    }
+    super.configure(req, res, next);
+  }
+
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    _.forEach(locals.items, i => {
+      const countryNumber = _.indexOf(locals.items, i);
+      const countryName = req.sessionModel.get('sih-countries').aggregatedValues;
+      _.forEach(i.fields, field => {
```

Comment:
You could include something like this to capitalise the value of the `is-risk-in-country` radio button group, in the summary page:
```
if (field.field === 'is-risk-in-country') {
  if (field.value.includes('yes')) {
    field.parsed = 'Yes';
  }
  else if (field.value.includes('no')) {
    field.parsed = 'No';
  }
}
```